daemons/pacemakerd/pcmkd_subdaemons.c

```diff
@@ -32,6 +32,7 @@ typedef struct pcmk_child_s {
     const char *command;
     const char *endpoint;  /* IPC server name */
     bool needs_cluster;
+    bool needs_retry;
 
     bool active_before_startup;
```

Comment:
FYI the idea behind the blank line is that everything after doesn't need to be statically initialized because it will be determined dynamically